src/Python/qsharp-core/qsharp/clients/iqsharp.py

```diff
@@ -261,7 +261,7 @@ def log_error(msg):
         handlers = {
             'execute_result': (lambda msg: results.append(msg)),
             'render_execution_path':  (lambda msg: results.append(msg)),
-            'display_data': lambda msg: ...
+            'display_data': display_data_handler
```

Comment:
I'm trying this change locally, and it appears to me that this makes `import qsharp` fail from Python.